parse/index.js

```diff
@@ -387,54 +388,63 @@ export default ({
           })
         }
 
-        propNode = {
-          type: 'Property',
-          loc,
-          name,
-          tags,
-          value: getValue(value),
-        }
+        if (name === 'format') {
+          block.format = getFormat(value)
+        } else {
```

Comment:
format is also a property @amymc so we should still be parsing it here but then we should skip it on the morphing process so it's not applied to the actual DOM/native elements themselves. I think the [isValidPropertyForBlock](https://github.com/viewstools/morph/blob/master/morph/react-dom/is-valid-property-for-block.js) functions are the place where that check could land and `format` would be false when it's used like this.